cnd/src/db.rs

```diff
@@ -197,11 +200,56 @@ pub struct CreatedSwap<A, B> {
     pub role: Role,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct Db<I>(pub I);
```

Comment:
Some rustdoc documentation would be nice for this one :)

Reply:
Good point, will add.